packages/react-reconciler/src/__tests__/ReactExpiration-test.internal.js

```diff
@@ -244,6 +244,16 @@ describe('ReactExpiration', () => {
       '1 [C] [render]',
       '1 [D] [render]',
     ]);
+
+    expect(Scheduler).toFlushAndYieldThrough([
+      '1 [A] [commit]',
+      '1 [B] [commit]',
+      '1 [C] [commit]',
+      '1 [D] [commit]',
+    ]);
+
+    // begin the update after the first one finished
+    expect(Scheduler).toFlushAndYieldThrough(['2 [A] [render]']);
```

Comment:
wouldn't this continue rendering through B, C, D, etc? or did you just want to add confirmations of the previous commits, and this render starting?

Reply:
Yea, it will render B, C, D but we don't need to write the redundant through because we know that will happen